lib/srv/termmanager.go

```diff
@@ -240,6 +268,7 @@ func (g *TermManager) CountRead() uint64 {
 }
 
 func (g *TermManager) Close() {
+	close(g.terminateNotifier)
```

Comment:
We still have the possibility of a double close panic here, right? Should probably atomically check `g.closed` and only close the channel the first time.

Reply:
Ah yeah, that's true. I was basing of assumption off this only getting called once but lets resolve that.